addons/l10n_in_ewaybill_irn/models/l10n_in_ewaybill.py

```diff
@@ -27,6 +27,21 @@ def _compute_is_process_through_irn(self):
                 )
             )
 
+    def _prepare_ewaybill_transportation_json_payload(self):
```

Comment:
@jva-odoo @hamo-odoo I might miss something, but I don't see any difference with the super method.

Reply:
Ok, now I saw in the commit msg that the keys are indeed differenT.